### PR TITLE
fix: dependabot-library-updates

### PR DESCRIPTION
**Pull request by jeffb-sfdc**
### What does this PR do?
This PR updates the following libraries:
commitlint/cli
commitlint/config-conventional
glob
path-exists
types/path-exists

...and makes the following PRs redundant:
https://github.com/forcedotcom/salesforcedx-vscode/pull/4174
https://github.com/forcedotcom/salesforcedx-vscode/pull/4249
https://github.com/forcedotcom/salesforcedx-vscode/pull/4390

### CODE REVIEW

File: packages/salesforcedx-vscode-apex-replay-debugger/src/index.ts [hunk 1]
```diff
@@ -243,14 +243,14 @@ function getDialogStartingPath(): vscode.Uri | undefined {
     const lastOpenedLogFolder = extContext.workspaceState.get<string>(
       LAST_OPENED_LOG_FOLDER_KEY
     );
-    if (lastOpenedLogFolder && pathExists.sync(lastOpenedLogFolder)) {
+    if (lastOpenedLogFolder && pathExists.pathExistsSync(lastOpenedLogFolder)) {
```
Comment (jeffb-sfdc) [inline packages/salesforcedx-vscode-apex-replay-debugger/src/index.ts:246]: With the update of the newer `pathExist` library, `sync()` was replaced with `pathExistSync()`

File: packages/salesforcedx-vscode-apex/test/vscode-integration/requirements.test.ts [hunk 2]
```diff
@@ -13,7 +13,11 @@ import { createSandbox, SinonSandbox, SinonStub } from 'sinon';
 import * as vscode from 'vscode';
 import { SET_JAVA_DOC_LINK } from '../../src/constants';
 import { nls } from '../../src/messages';
-import { checkJavaVersion, JAVA_HOME_KEY, resolveRequirements } from '../../src/requirements';
+import {
```
Comment (jeffb-sfdc) [inline packages/salesforcedx-vscode-apex/test/vscode-integration/requirements.test.ts:16]: Auto-formatting that took placed when i saved the file.

File: packages/salesforcedx-vscode-apex/test/vscode-integration/requirements.test.ts [hunk 3]
```diff
@@ -69,7 +73,9 @@ describe('Java Requirements Test', () => {
       const result = await checkJavaVersion('~/java_home');
       expect(result).to.equal(true);
     } catch (err) {
-      fail(`Should not have thrown when the Java version is 17.  The error was: ${err}`);
+      fail(
```
Comment (jeffb-sfdc) [inline packages/salesforcedx-vscode-apex/test/vscode-integration/requirements.test.ts:76]: more auto-formatting

**Review by gbockus-sf — APPROVED** [review]
🍠 🚢

**Pull request merged**